tests/system/small/test_session.py

```diff
@@ -369,6 +369,17 @@ def test_read_pandas(session, scalars_dfs):
     pd.testing.assert_frame_equal(result, expected)
 
 
+def test_read_pandas_inline_respects_location():
+    options = bigframes.BigQueryOptions(location="europe-west1")
+    session = bigframes.Session(options)
+
+    df = session.read_pandas(pd.DataFrame([[1, 2, 3], [4, 5, 6]]))
```

Comment:
Could we add a test for inline data too, and verify that it creates the result tables in the intended location? `bpd.Dataframe([[1, 2, 3], [4, 5, 6]])`

Reply:
added